### PR TITLE
Bug 1492245 - Clean up Focus signing servers config

### PR DESCRIPTION
**Pull request by JohanLorenzo**

### CODE REVIEW

File: modules/signing_scriptworker/templates/passwords-mobile.json.erb [hunk 1]
```diff
@@ -1,11 +1,5 @@
...
-        ["signing9.srv.releng.mdc1.mozilla.com:9120", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_release_password"]) %>", ["focus-jar"], "signing_server"],
-        ["signing10.srv.releng.mdc2.mozilla.com:9120", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_release_password"]) %>", ["focus-jar"], "signing_server"],
-        ["signing11.srv.releng.mdc2.mozilla.com:9120", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_release_password"]) %>", ["focus-jar"], "signing_server"],
-        ["signing12.srv.releng.mdc2.mozilla.com:9120", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_release_password"]) %>", ["focus-jar"], "signing_server"],
         ["https://autograph-external.prod.autograph.services.mozaws.net", "<%= scope.function_secret(["autograph_focus_prod_username"]) %>", "<%= scope.function_secret(["autograph_focus_prod_password"]) %>", ["autograph_focus"], "autograph"]
```
Comment (MihaiTabara) [inline modules/signing_scriptworker/templates/passwords-mobile.json.erb]: Should we change to `autograph_focus` at some point for consistency?

File: modules/signing_scriptworker/templates/passwords-mobile.json.erb [hunk 2]
```diff
@@ -1,11 +1,5 @@
 {
     "project:mobile:focus:releng:signing:cert:release-signing": [
-        ["signing7.srv.releng.mdc1.mozilla.com:9120", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_release_password"]) %>", ["focus-jar"], "signing_server"],
```
Comment (MihaiTabara) [inline modules/signing_scriptworker/templates/passwords-mobile.json.erb]: Yes please! One more train to ride and we can turn off Fennec as well \o/

File: modules/signing_scriptworker/templates/passwords-mobile.json.erb (hunk 1 above)
Comment (JohanLorenzo) [reply modules/signing_scriptworker/templates/passwords-mobile.json.erb]: Yeah, I was thinking of this too. As this is not a trivial change to make, I'd defer it for after the holidays. I don't want to break the releases, if one occurs while one of us is out.

Comment (MihaiTabara) [reply modules/signing_scriptworker/templates/passwords-mobile.json.erb]: This makes perfect sense. Let's address it in Q1 with fresh 👀

**Pull request merged**